apps/python/src/notifier/markdown.py

```
@@ -0,0 +1,225 @@
+        "type": "table",
+        "table": {
+            "table_width": table_width,
+            "has_column_header": has_header,
+            "has_row_header": False,
+            "children": children,
+        },
+    }
+
+
+def _split_label_colon(line: str) -> list[str]:
+    """「ラベル：内容」形式の行を2行に分割して返す。該当しない場合はそのまま返す。
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)